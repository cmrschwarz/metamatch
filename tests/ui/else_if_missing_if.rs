use metamatch::template;

fn main() {
    let res = template! {
        [<if 3 > 2>]
        true
        // missing `if` below.
        // the current error for this is not ideal but we certainly
        // don't want it do regress further
        [<else x > 2>]
        false
        [</if>]
    };

    assert!(res);
}
