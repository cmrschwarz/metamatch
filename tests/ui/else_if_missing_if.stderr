error: unmatched closing tag `/if`
  --> tests/ui/else_if_missing_if.rs:12:9
   |
12 |         [</if>]
   |         ^^^^^^^
